tests/tt_eager/python_api_testing/unit_testing/misc/test_transpose.py

```diff
@@ -747,17 +745,20 @@ def test_transpose_4d_wh_tile(shape, device):
     assert_with_pcc(torch_output, tt_output, 0.9999)
 
 
+@pytest.mark.skip("Skipping due to hang on to_layout to tile where input shape has 1 in it")
 @pytest.mark.parametrize(
     "config",
     [
-        [[1, 1370, 1, 3, 1280], [0, -2], ttnn.TILE_LAYOUT],  # untilize doesn't work with 4D
-        [[1, 50, 1, 3, 768], [0, -2], ttnn.TILE_LAYOUT],  # untilize doesn't work with 4D
-        [[21843, 768], [0, 1], ttnn.ROW_MAJOR_LAYOUT],  # circular buffer overflow
+        [[1, 1370, 1, 3, 1280], [0, -2], ttnn.TILE_LAYOUT],  # hang
+        [[1, 50, 1, 3, 768], [0, -2], ttnn.TILE_LAYOUT],  # hang
+        [[1, 50, 1, 3, 1024], [0, -2], ttnn.TILE_LAYOUT],  # hang
+        [[1, 197, 1, 3, 768], [0, -2], ttnn.TILE_LAYOUT],  # hang
+        [[1, 197, 1, 3, 1024], [0, -2], ttnn.TILE_LAYOUT],  # hang
+        [[2, 7, 2, 7, 384], [-4, -3], ttnn.TILE_LAYOUT],  # hang
     ],
 )
-@pytest.mark.parametrize("memory_config", [ttnn.L1_MEMORY_CONFIG, ttnn.DRAM_MEMORY_CONFIG])
+@pytest.mark.parametrize("memory_config", [ttnn.DRAM_MEMORY_CONFIG])
```

Comment:
Why remove L1 memory configs?

Reply:
An accident while testing, which was especially dumb because this is a disabled test.